controllers/front/actions.php

```diff
@@ -77,7 +77,7 @@ public function processRemove()
             die('0');
         }
 
-        die(1);
+        die('1');
```

Comment:
I confess that I don't understand why in the first condition we `die('0')` if something went wrong, then `die('0')` if the mail alert was successfully deleted, and to finish, `die(1)` :thinking:  Do you know where it's used?

Reply:
Fixed